Webhook throws an error if the endpoint returns a 2xx status code != 200

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior
If the webhook endpoint returns a 2xx status code unequal to 200, the sending event should be treated as success.

<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->

## Current Behavior
The request fails for a 201 response:
```
[2018-04-04T15:35:05.933883248Z] [ERROR] unable to send event s3:ObjectAccessed:Get of bucket: [redacted], object: [redacted] to target 1:webhook (sending event failed with 201 Created)
```
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->

## Possible Solution
Treat all 2xx status codes as success.
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Configure minio to send notifications to a webhook endpoint that returns a 20x status code different from 200.
2. Set up notification for a resource
3. See the error in the minio log

## Context
This happens while sending webhook notifications to a publish/subscribe service based on nginx and nchan, which returns several 20x status codes for success. Other webhook sources accept these status codes as success, e.g., GDrive and Dropbox.
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used (`minio version`):
```
# minio version
Version: 2018-03-30T00:38:44Z
Release-Tag: RELEASE.2018-03-30T00-38-44Z
Commit-ID: 804a4f9c159ecef7093585a5b577e03a382af50e
```

--- a/pkg/event/target/webhook.go
+++ b/pkg/event/target/webhook.go
@@ -79,12 +79,11 @@ func (target *WebhookTarget) Send(eventData event.Event) error {
 	// FIXME: log returned error. ignore time being.
 	_ = resp.Body.Close()
 
-	switch resp.StatusCode {
-	case http.StatusOK, http.StatusAccepted, http.StatusContinue:
-		return nil
-	default:
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
 		return fmt.Errorf("sending event failed with %v", resp.Status)
 	}
+
+	return nil
 }
 
 // Close - does nothing and available for interface compatibility.

